android/src/main/java/com/mybigday/rnmediameta/RNMediaMeta.java

```diff
@@ -78,7 +79,18 @@ private void putString(WritableMap map, String key, String value) {
     if (value != null) map.putString(key, value);
   }
 
-  private void getMetadata(String path, Promise promise) {
+  private void getMetadata(String pathOrJSONString, Promise promise) {
```

Comment:
It will just change `get(path)` to `get(path, options)` for the JS function (we also need to check `options` not null), so I don't think it make a breaking change.